argh_derive/src/lib.rs

```diff
@@ -291,7 +291,18 @@ fn impl_from_args_struct(
                     } else {
                         __remaining_args
                     };
-                    #name = Some(<#ty as argh::FromArgs>::from_args(&__command, __remaining_args)?);
+                    if __dump_args {
```

Comment:
This command seems pretty specialized. Could we make this an opt-in feature, with something like:

```
#[derive(FromArgs)]
#[argh(description = "", dump_passed_args)]
struct Cmd { ... }
```

Happy for suggestions on a better attribute name. We could even make it such that this flag was customizable, with something like:

```
#[derive(FromArgs)]
#[argh(description = "", dump_passed_args="--dump-passed-args")]
struct Cmd { ... }
```

To make it easier for people to customize.

Reply:
See above question about a TopLevel opt-in versus per subcommand. 

Is the benefit that the macro expansion will not generate the bits of logic to accumulate the passed_args if the user has opted out?